eva-server/src/test/java/uk/ac/ebi/eva/server/ws/VariantWSServerTest.java

```diff
@@ -172,4 +172,22 @@ private Boolean testCheckVariantExistsHelper(String testIdRegion) throws URISynt
         return results.get(0);
     }
 
+    @Test
+    public void testCountTotalNumberOfVariantsService() throws URISyntaxException {
+        testCountTotalNumberOfVariantsServiceHelper(new Long(0));
+    }
+
+    private void testCountTotalNumberOfVariantsServiceHelper(Long expectedVariantsCount) throws URISyntaxException {
```

Comment:
Given that this is only used by one method (and will probably ever be) I would merge this into the test method or at least change its name to `testCountTotalNumberOfVariantsHelper`.